src/components/Filter.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import {nanoid} from 'nanoid'
+
+const regions = ['All', 'Africa', 'Americas', 'Asia', 'Europe', 'Oceania'];
+
+const Filter = ({
+  setSearchValue,
+  searchValue,
+  setSelectedRegion,
+  selectedRegion,
+}) => {
+  const [openDropDown, setOpenDropDown] = useState(false);
+  return (
+    <>
+      <section className='filters'>
+        <div className='container'>
+          <div className='search-wrapper'>
+            <i className='fa-regular fa-magnifying-glass search-icon'></i>
+            <input
+              type='text'
+              className='search-input'
+              placeholder='Search for a country...'
+              value={searchValue}
+              onChange={(e) => setSearchValue(e.target.value)}
+            />
+          </div>
+
+          <div
+            className='dropdown-wrapper'
+            onClick={() => setOpenDropDown(!openDropDown)}
+          >
```

Comment:
Usually when we are using the `!` operator on value we would call the function `toggleSomething`